src/redux/root.ts

```diff
@@ -4,17 +4,27 @@ import { Reducer, combineReducers } from 'redux';
 import { LocationState, locationStateReducer } from './location/reducer';
 import { CallState, callStateReducer } from './callState/reducer';
 import { RemoteDataState, remoteDataReducer } from './remoteData/reducer';
+import { UserStatsState, userStatsReducer } from './userStats/reducer';
+
+export type OutcomeType =
```

Comment:
Can we match these to the outcome models from the server? Even with custom outcomes, local impact will always map back to `unavailable`,  `voicemail` or `contact`. I don't believe we counted skips previously and that is fine to maintain.

Reply:
makes sense.